### PR TITLE
[JENKINS-40296] Tests to cover changeset messages

### PR DESCRIPTION
**Pull request by imeredith**
# Description

See [JENKINS-40296](https://issues.jenkins-ci.org/browse/JENKINS-40296).

BO PR https://github.com/jenkinsci/blueocean-plugin/pull/651
# Submitter checklist
- [ ] Link to JIRA ticket in description, if appropriate.
- [ ] Change is code complete and matches issue description
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.
- [ ] Ran Acceptance Test Harness against PR changes.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

@reviewbybees

### CODE REVIEW

**Comment by michaelneale** [comment]
:bee:

File: src/main/js/api/git.js [hunk 1]
```diff
@@ -127,15 +127,55 @@ exports.createBranch = function (branchName, pathToRepo) {
         });
 };
 
+/** 
+ * Creates a file and adds it to repo.
+ * 
+ * FIXME: doesnt seem to actally git add the file, altohugh it does create the commit hash.
```
Comment (sophistifunk) [inline src/main/js/api/git.js]: Typo

**Comment by sophistifunk** [comment]
I haven't had much hands-on with ATH but this all looks sensible to me. 🐝

File: src/main/js/api/git.js (hunk 1 above)
Comment (michaelneale) [inline src/main/js/api/git.js]: actally is a word. Actually.

**Pull request merged**